agent/plugins/syntests/httpPing/README.md

```diff
@@ -17,8 +17,21 @@ Pings a HTTP endpoint and checks if its the expected response.
     timeoutRetries: 1
 ```
 
+```yaml
+  config : |
+    address: "localhost:9200"
+    expectedCodeRegex : ^(202|472){1}
+    consecutiveTestSuccessCount: 3
+    consecutiveTestInterval: 3s
+```
+
 `retries` is max retry times when http request failed except exceeded timeout
 `timeoutRetries` is optional, default value is 0. Max retry times only when http request exceeded timeout. Recommended value is small number, like 1.
+`retries` and `timeoutRetries` retry the ping test until it http request succeed or reach the max retry times.
+
+`consecutiveTestSuccessCount` is the total http ping test times. If any failure occurs, the final test result will be failed. It conflicts with `retries` and `timeoutRetries`.
+`consecutiveTestInterval` is ping interval in the http ping test. Default value is 5s.
+`retries`/`timeoutRetries` and `consecutiveTestSuccessCount` are mutually exclusive. They can't be larger than 0 at the same time.
```

Comment:
Hey @lunzhou2024 
Can you please rewrite this readme. Its a little difficult to understand the different options. Maybe create a table with the options. 

For `consecutiveTestSuccessCount` - can we rename it to `repeatsWithoutFail` - Number of repeats of the test, without tolerating any failure, `retries` and `timeoutRetries` options are ignored
and for `consecutiveTestInterval` can we rename it to - 'waitBetweenRepeats' - Time to wait between the repeat